CONTRIBUTING.md

```diff
@@ -1,36 +1,38 @@
-# Contributing to aframe
+# Contributing to Aframe
 
 Thanks for taking the time to contribute! While we appreciate all who wish to contribute, at the moment we're prioritizing contributions from IGWN scientists.
 
 Here are some guidelines on how code in this repo should be structured, as well as best practices for making contributions to it.
 
 ## Getting started
-To start contributing, create a [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this repository. You can then either clone your fork
+> **Note** It is assumed that you have succesfully completed the [ml4gw quickstart](https://github.com/ml4gw/quickstart/) instructions for pre-requisite environment setup and software installation.
+
+To start contributing, create a [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this repository. You can then clone your fork
 
 ```console
-git clone git@github.com:johnny-tsunami/aframe.git
```

Comment:
RIP Johnny. Good for LVK consistency though

Reply:
Hahah sorry Alec